promptsource/templates/piqa/templates.yaml

```diff
@@ -129,9 +195,11 @@ templates:
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
     name: 'Correct the solution if false: from sol 2'
```

Comment:
Need something in the template to instruct the model to re-copy if the solution makes sense